src/renderer/lib/validate.ts

```diff
@@ -23,10 +23,10 @@ export const isValidIpAndPort = (ip: string) => {
     return "Invalid Port";
   }
 
-  return isValidIpAddress(ip);
+  return validateIpAddress(ip);
```

Comment:
```suggestion
  if (!isValidIpAddress(ip)) {
    return "Invalid IP address";
  }
  
  return true;
```

For completeness I think we should do this.